### PR TITLE
docs(read-only): slider docs

### LINKED ISSUE #3269: [Read-only inputs]: Slider website docs

### PR DESCRIPTION
**Pull request by laurenmrice**
Closes #3269 

----

**New**

Usage tab:
- Added Overview section in context image.
- Added Behaviors and States sections with content and image.

Style tab:
- Added Color section image.
- Updated Interactive states table content and added image.
- Update Structure section spec image.

### CODE REVIEW

File: src/data/guidelines/color-tokens.js [hunk 1]
```diff
@@ -881,8 +881,8 @@ const colorTokens = {
           hex: '#e0e0e0',
         },
         g10: {
-          name: 'Gray 30',
```
Comment (tw15egan) [inline src/data/guidelines/color-tokens.js:884]: Do we want to be changing this / removing the `border-tile` tokens?

Comment (laurenmrice) [reply src/data/guidelines/color-tokens.js:884]: 😓 None of these changes are supposed to be in this PR. Only the usage.mdx, style.mdx, and images for `Slider` have been changed in this PR. I am not sure why this keeps happening.

Comment (tw15egan) [reply src/data/guidelines/color-tokens.js:884]: Okay I'll remove those changes

Comment (tw15egan) [reply src/data/guidelines/color-tokens.js]: Should be good now, let me know if everything looks correct

Comment (laurenmrice) [reply src/data/guidelines/color-tokens.js]: Yes! This all looks correct now. Thank you!

**Review by tw15egan — APPROVED** [review]

**Review by sstrubberg — APPROVED** [review]

**Pull request merged**